### PR TITLE
Create test-7.js

### PR DESCRIPTION
**Pull request by daint-1849**

### CODE REVIEW

**Comment by daint-1849** [comment]
123

**Review by daint-1849 — COMMENTED** [review]
123

**Comment by daint-1849** [comment]
dai comment

**Review by daint-1849 — COMMENTED** [review]
ok

**Comment by daint-1849** [comment]
123

**Review by daint-1849 — COMMENTED** [review]
123

File: examples/test-7.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log('Test 7');
```
Comment (daint2git) [inline examples/test-7.js:1]: 123

**Review by daint2git — DISMISSED** [review]
123

File: examples/test-7.js (hunk 1 above)
Comment (daint-1849) [inline examples/test-7.js:1]: 123

**Review by daint-1849 — COMMENTED** [review]
123123

File: examples/test-7.js (hunk 1 above)
Comment (daint2git) [reply examples/test-7.js:1]: 123

**Review by daint2git — CHANGES_REQUESTED** [review]

**Review by daint-1849 — COMMENTED** [review]
123345

**Review by daint2git — APPROVED** [review]

File: examples/test-7.js (hunk 1 above)
Comment (daint2git) [reply examples/test-7.js:1]: aaa

**Review by daint2git — CHANGES_REQUESTED** [review]

**Review by daint-1849 — COMMENTED** [review]
123

**Review by daint2git — CHANGES_REQUESTED** [review]
123

**Comment by daint-1849** [comment]
123

**Review by daint-1849 — COMMENTED** [review]
hihi

**Comment by daint-1849** [comment]
abc

**Review by daint-1849 — COMMENTED** [review]
123

**Review by daint2git — APPROVED** [review]

**Comment by daint2git** [comment]
abc

**Comment by daint-1849** [comment]
123

**Comment by daint-1849** [comment]
123

**Review by daint2git — COMMENTED** [review]
123

**Comment by daint-1849** [comment]
ola ola

File: examples/test-7.js (hunk 1 above)
Comment (daint2git) [reply examples/test-7.js:1]: mmmm

**Review by daint2git — APPROVED** [review]

**Review by daint2git — CHANGES_REQUESTED** [review]
on thong

**Comment by daint2git** [comment]
123

**Review by daint2git — APPROVED** [review]

**Pull request merged**